client/src/context/useAuthContext.tsx

```diff
@@ -46,7 +59,9 @@ export const AuthProvider: FunctionComponent = ({ children }): JSX.Element => {
       await loginWithCookies().then((data: AuthApiData) => {
         if (data.success) {
           updateLoginContext(data.success);
-          history.push('/dashboard');
+          if (profileData) {
+            history.push(`/edit-profile/${profileData.profileId}`);
```

Comment:
Similar to above, if the user already has a profile why are we redirecting them to the 'edit-profile' page?